docs/plugins/integrations/sync.rst

```diff
@@ -0,0 +1,77 @@
+**************************
+Integration Sync Engine
+**************************
+
+.. contents:: Table of Contents
+
+The sync engine supports bidirectional syncing between Mautic's contact and companies with third party objects. The engine generates a "sync report" from Mautic that it converts to a "sync order" for the integration to process. It then asks for a "sync report" from the integration which it converts to a "sync order" for Mautic to process.
+
+When building the report, Mautic or the integration will fetch the objects that have been modified or created within the specified timeframe. If the integration supports changes at the field level, it should tell the report on a per field basis when the field was last updated. Otherwise, it should tell the report when the object itself was last modified. These dates are used by the "sync judge" to determine which value should be used in a bi-directional sync.
```

Comment:
```suggestion
When building the report, Mautic or the Integration will fetch the objects that have been modified or created within the specified timeframe. If the Integration supports changes at the field level, it should tell the report on a per field basis when the field was last updated. Otherwise, it should tell the report when the object itself was last modified. These dates are used by the "sync judge" to determine which value should be used in a bi-directional sync.
```